src/inRange.js

```diff
@@ -0,0 +1,35 @@
+import { ifElse, curry, useWith, flip, both, gte, lt } from 'ramda';
+
+const errorMessage = `low must not be greater than high in inRange(low, high, value)`;
+
+/**
+ * Checks if input value falls within the supplied range. The bottom of the range is inclusive, the
+ * top of the range is exclusive.
+ *
+ * @func inRange
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.6.0|v2.6.0}
+ * @category Relation
+ * @sig Number -> Number -> Number -> Boolean
+ * @param {Number} low The lowest value within the range
+ * @param {Number} high The first value outside the range
+ * @param {Number} value The value to test
+ * @return {Boolean}
+ * @throws {Error} When `low` is greater than or equal to `high`
+ * @example
+ *
+ * RA.inRange(0, 5, 3); //=> true
+ * RA.inRange(0, 5, 0); //=> true
+ * RA.inRange(0, 5, 4); //=> true
+ * RA.inRange(0, 5, 5); //=> false
+ * RA.inRange(0, 5, -1); //=> false
+ */
+export default curry((low, high, value) =>
+  ifElse(
+    gte,
+    () => {
+      throw new Error(errorMessage);
+    },
+    useWith(both, [flip(gte), flip(lt)])
+  )(low, high)(value)
+);
```

Comment:
I'd extract the implementation so that we don't have to compose in every composition.

```js
const inRangeImpl = ifElse(
  gte,
  () => {
    throw new Error('low must not be greater than high in inRange(low, high, value)');
  },
  useWith(both, [flip(gte), flip(lt)])
);

const inRange = curry((low, high, value) => inRangeImpl(low, high)(value));
```

Also I'm looking for some translation of `flip(gte)` into some function that is already in R without composition..isn't it true that `flip(gte)` is equivalent to `lt` ? I'm not sure if the logic is found, is early morning ;]

Reply:
What do you mean by

> we don't have to compose in every run

Reply:
> isn't it true that flip(gte) is equivalent to lt

No, but it is equivalent to `lte`, and `flip(lt)` is equivalent to `gt`. I find the order of args really hard to reason about, but it makes sense to avoid the `flip`.

Reply:
> What do you mean by

`ifElse` is HOC. In every call to `inRange` the `ifElse` has to always be composed. If you extract it outside, it is composed once, and used in `inRange` function. Does it make sense ?

Reply:
Thanks. Yes it makes sense.